CODE_OF_CONDUCT.md

```diff
@@ -47,10 +47,10 @@ communication. It applies to all of your communication and conduct in these
 spaces, including emails, chats, things you say, slides, videos, posters, signs,
 or even t-shirts you display in these spaces.
 
-All community members should help support our standards of acceptable behavior.
-Everyone is encouraged to speak up in response to any behavior that they deem
-inappropriate, threatening, offensive, or harmful. If you believe someone is
-violating the code of conduct, please report it to the
+- All community members should help support our standards of acceptable behavior.
+- Everyone is encouraged to speak up in response to any behavior that they deem
```

Comment:
I see some stray `-` got included here

Reply:
its gonna make any difference? my preview window shows it alright?

Reply:
after your most recent formatting this is fine.